modules/ROOT/pages/cryptography-pgp.adoc

```diff
@@ -593,7 +601,8 @@ Binary
 |===
 | Field | Type | Description | Default Value | Required
```

Comment:
I looked at this table in output and noticed that the right two columns are underutilized.
I wonder if we might replace these tables, which include a lot of narrative, with definition lists in which the list item is the field and type and the definition includes the default value and requirement?
However, if your audience wants to compare/contrast the fields, table format might be necessary.
See the style ref. entry for tables for further guidance.